fhservices.cfg.j2

```diff
@@ -10,6 +10,12 @@ define command {
        command_line   /opt/rhmap/nagios/plugins/fh-check-component-health.sh $ARG1$ 8080
 }
 
+# Check for valid response from Memcached
+define command {
+       command_name   check_memcached
+       command_line   $USER1$/check_tcp -H memcached -p $ARG1$ -E -s 'version\n' -e 'VERSION' -w2 -c5 -t5
```

Comment:
Why is critical == timeout?

The explanation gives the general idea, thanks, but what does `-E -s 'version\n' -e 'VERSION'` do?